src/i18n/en-US/home/settings.ts

```diff
@@ -0,0 +1,64 @@
+const homepageSettings = {
+  heading: 'Homepage Settings',
+  stepOne: '(Step 1 of 2)',
+  stepTwo: '(Step 2 of 2)',
+  description:
+    'Display the content that’s most meaningful to your role. You can update this at any time.',
+  descriptionTwo:
+    'Arrange the content however you wish. You can update this at any time.',
+  selection: 'Select which information to display on your homepage.',
+  selectionTwo: 'Set the display order for your homepage.',
+  settings: {
+    MY_MODEL_PLANS: {
```

Comment:
Is there any way we can type this `settings` object easily? It'd be nice to ensure we're covering all cases/using the enums from the generated GQL for these

If not, definitely not a dealbreaker!